pkg/cmd/roachprod/flags.go

```diff
@@ -490,4 +491,6 @@ func initFlags() {
 		"time-range", []int64{}, "grafana annotation time range in epoch time")
 	fetchLogsCmd.Flags().DurationVarP(&fetchLogsTimeout,
 		"timeout", "t", 5*time.Minute, "Timeout for fetching the logs from the cluster nodes")
+	pprofTimeBefore = getLatestPprof.Flags().StringP(
+		"time-before", "t", "", "Timeout for fetching the logs from the cluster nodes")
```

Comment:
Forgot to update the description? I'd also use `Duration` or at least hint at what time units are expected.